### PR TITLE
feat(theme): Add Notifications components

### PR DESCRIPTION
**Pull request by Danktuary**
**Please describe the changes this PR makes and why it should be merged:**
This adds a couple new components to display notifications at the bottom of the screen. Right now, this will be used to let users know that they're on the `v13-prep` branch while it's still hosted on the `v13-prep.discordjs.guide` subdomain. Once the branch gets merged into `main`, this notification can get updated, and a similar one will be added on the v12/v11 branch.

Dismissing a notification saves to `localStorage` and won't be shown again once dismissed.

Screenshot: notifications components

### CODE REVIEW

**Comment by J-Human** [comment]
Looks good, but just a simple request - could the message color be a bit darker on light mode because it might be hard to read for other people who can't see properly?

3D8ED1A9-F347-4994-A5C7-EE058F15E7AC

**Comment by Danktuary** [comment]
> Looks good, but just a simple request - could the message color be a bit darker on light mode because it might be hard to read for other people who can't see properly?
> 
> 3D8ED1A9-F347-4994-A5C7-EE058F15E7AC

Whoops, forgot about light theme. Fixed. 👍🏼

**Review by J-Human — APPROVED** [review]

File: guide/.vuepress/theme/components/icons/Close.vue [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+<template>
+	<svg viewBox="0 0 24 24"><path d="M19 6.41L17.59 5L12 10.59L6.41 5L5 6.41L10.59 12L5 17.59L6.41 19L12 13.41L17.59 19L19 17.59L13.41 12L19 6.41z" fill="currentColor"></path></svg>
```
Comment (WiseDevHelper) [inline guide/.vuepress/theme/components/icons/Close.vue:3]:
```suggestion
	<svg viewBox="0 0 24 24"><path d="M19 6.41L17.59 5L12 10.59L6.41 5L5 6.41L10.59 12L5 17.59L6.41 19L12 13.41L17.59 19L19 17.59L13.41 12L19 6.41z" fill="currentColor" /></svg>
```

**Review by WiseDevHelper — CHANGES_REQUESTED** [review]
LGTM otherwise! 🎉

**Pull request merged**